js/index.js

```diff
@@ -17,22 +21,95 @@ const skills = [
     'Engineering', 
     'Reserch'
 ];
+//Fill out a Skill section with skills from the skills array. 
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul');
 for(let i = 0; i < skills.length; i++) {
     let skill = document.createElement('li');
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 }
-const listItems = skillsSection.getElementsByTagName('li');
+//Add an event to the Skill section
+/* const listItems = skillsSection.getElementsByTagName('li');
 for(let i = 0; i < listItems.length; i++) {
     listItems[i].addEventListener('mouseover', () => {
        listItems[i].textContent = listItems[i].textContent.toUpperCase()
     })
     listItems[i].addEventListener('mouseout', () => {
        listItems[i].textContent = skills[i];
     })
+} */
+
+//Add an event to the Skill section using Bubbling and event Delegation
+skillsList.addEventListener('mouseover', (event) => {
+    if (event.target.tagName == 'LI') {
+        event.target.textContent = event.target.textContent.toUpperCase();
+  }
+});
+skillsList.addEventListener('mouseout', (event) => {
+    if(event.target.tagName == 'LI') {
+      for(let i = 0; i < skills.length; i++) {
+          if(event.target.textContent == skills[i].toUpperCase()) {
+            event.target.textContent = skills[i]
+          }
+        }
+    }
+});
+//make message section invisible by default    
+messageSection.style.display = 'none'
+//function to create a button
+function createButton(nameButton) {
+    const button = document.createElement('button')
+    button.innerText = nameButton
+    button.type = 'button'
+    return button
 }
+//Working with Message Section: filling it in with messages from th users
+messageForm.addEventListener('submit', (event) => {
```

Comment:
Make sure you're giving proper spacing on all of your code. It is slightly difficult to read through all the code. It all works however that isn't our only goal. You'll notice this as you come back to work again on your Portfolio over time. You'll look for specific files and wonder where they all live.